src/compoz.test.ts

```diff
@@ -97,25 +97,25 @@ link.
 }
 
 function createFullCompose() {
-	const files: ICompozFile[] = []
+	// const files: ICompozFile[] = []
```

Comment:
Kenapa di-komen?

Reply:
ini buat ngeliat tampilannya biar file attachmentnya ga muncul kak pas di test compoznya, makanya kemarin di comment dulu

Reply:
Either delete the commented code or un-comment them back.